.changesets/feat_bnjjj_fix_5930.md

```diff
@@ -0,0 +1,20 @@
+### Add ability to alias standard attributes for telemetry ([Issue #5930](https://github.com/apollographql/router/issues/5930))
+
+There is an issue when using standard attributes (on cache for example) because on new relic `entity.type` is a reserved attribute name and so it won’t work properly. cf [Learn about New Relic entities](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#reserved-attributes)  Moreover `entity.type` is not consistent with our other graphql attributes (prefixed by `graphql.`). So we rename `entity.type` attribute to `graphql.type.name`.
+
+In order to make it work and that could also answer other use cases that would be great if we can alias the name of a standard attribute like this:
```

Comment:
```suggestion
### Support aliasing standard attributes for telemetry ([Issue #5930](https://github.com/apollographql/router/issues/5930))

The router now supports creating aliases for standard attributes for telemetry.

This fixes issues where standard attribute names collide with reserved attribute names. For example, the standard attribute name `entity.type` is a [reserved attribute](New Relic entities](https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/what-entity-new-relic/#reserved-attributes) name for New Relic, so it won't work properly. Moreover `entity.type` is inconsistent with our other GraphQL attributes prefixed with `graphql.` 

The example configuration below renames `entity.type` to `graphql.type.name`:
```

Reply:
@bnjjj  By this evening, could you please review this editorialized content for the changelog by tomorrow and "Commit suggestion" if it's good?  If it needs slight clarifications or edits, please just tweak @shorgi's comment using the "Edit" button on the suggestion and commit that edited suggestion.  Thanks!